orchestration/dagster_orchestration/hca_manage/validation.py

```diff
@@ -59,6 +60,23 @@ def validate_directory(path: str, bucket: storage.Client.bucket) -> None:
         logging.info('File path and Json are valid')
 
 
+def validate_staging_area(path: str) -> None:
+    """
+    Run the UCSC pre-checks on the staging area to identify potential snapshot or indexing failures
+    :param path: Google stage path name
+    """
+    adapter = StagingAreaValidator(
+        staging_area=path,
+        ignore_dangling_inputs=False,
```

Comment:
🔧 This should be passed in as a command line arg and threaded through, there will likely be situations where we want to set the to `True`

Reply:
That makes sense, I refactored the code to take in an "ignore_inputs" argument for this. I tried rerunning the validator with a "True" argument on the staging area I was using for testing (gs://broad-dsp-monster-hca-dev-test-storage/integration/ebi_small/test_data) but I was getting the same result as before. I don't think this is an issue because I don't think the errors the validator throws on the testing area area related to dangling inputs that should be ignored - right?